### ISSUE TITLE
False alarms with `require-hook`, `no-done-callback`, and `no-standalone-expect`

### ISSUE THREAD

**Issue by mikestopcontinues** [veritem/eslint-plugin-vitest#182]
Here's some basic false alarms:

```ts
// eslint-disable-next-line vitest/require-hook
describe.skipIf(true)('suite', () => {
  test('test', async () => {
    expect(true).toBeTruthy();
  });
});

// eslint-disable-next-line vitest/require-hook
describe.concurrent('suite', () => {
  test('test', async () => {
    expect(true).toBeTruthy();
  });
});
```

Here's a larger example with three different false alarms (`require-hook`, `no-done-callback`, and `no-standalone-expect`):

```ts
import type {TestAPI, SuiteAPI} from 'vitest';
import {beforeEach, describe, test, expect} from 'vitest';

interface Context {
  value: string;
}

type ContextScope = {
  test: TestAPI<Context>;
  describe: SuiteAPI<Context>;
};

type ContextScopeFn = (scope: ContextScope) => void;

const withContext = (fn: ContextScopeFn) => {
  // eslint-disable-next-line vitest/no-done-callback
  beforeEach<Context>((ctx) => {
    ctx.value = Math.random();
  });

  fn({
    test: test as TestAPI<Context>,
    describe: describe as SuiteAPI<Context>,
  });
};

// eslint-disable-next-line vitest/require-hook
withContext(({describe, test}) => {
  describe('scraping', () => {
    test('it scrapes', async ({value}) => {
      // eslint-disable-next-line vitest/no-standalone-expect
      expect(value).toBeTruthy();
    });
  });
});

```

**Comment by SalahAdDin** [comment]
We are in September and we still have this issue:
![image](https://github.com/veritem/eslint-plugin-vitest/assets/5034215/96ac3bd8-0e71-44e4-8afc-6af0299d3245)

**Comment by veritem** [comment]
@SalahAdDin I have so much todo with less time. I'll hopefully fix this sooner

**Comment by SalahAdDin** [comment]
I think it was our mistake: I forgot to add the **types** to `tscofing` file, after doing this, it seems all the false alarms disappeared.

**Comment by veritem** [comment]
@SalahAdDin  if that's the case I'll go  a head and close this issue. If it happens in future you can reopen again.
